SmartcarAuth/VehicleInfo.swift

```diff
@@ -21,15 +21,20 @@
  */
 
 @objc public class VehicleInfo: NSObject {
+    var vin: String?
     var make: String?
-    
+    var model: String?
+    var year: Int? 
     /**
      Constructor for VehicleInfo
      - parameters:
      - make: allows user to bypass OEM selection screen and go directly to vehicle login screen, defaults to nil
```

Comment:
Since this is essentially a type definition, it should not describe where it might be used. This is the same thing we talked about with Jack regarding the Android SDK's VehicleInfo type-def.

Reply:
oops, removed